mu4/scenes/notation/internal/scenenotationconfiguration.cpp

```diff
@@ -86,6 +86,13 @@ Channel<QColor> SceneNotationConfiguration::foregroundColorChanged() const
     return m_foregroundColorChanged;
 }
 
+QColor SceneNotationConfiguration::playbackCursorColor() const
+{
+    QColor c("#ff0000");
```

Comment:
Just a color code? The color will not depend on the chosen theme?

Reply:
Added todo comment